modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/consented/token/OAuth2ServiceAuthCodeGrantJWTAccessTokenWithConsentedTokenColumnTestCase.java

```diff
@@ -111,9 +111,9 @@ public void testSendAuthorizedPost() throws Exception {
         urlParameters.add(new BasicNameValuePair("authorizeEndpoint", OAuth2Constant.APPROVAL_URL));
         urlParameters.add(new BasicNameValuePair("authorize", OAuth2Constant.AUTHORIZE_PARAM));
         // email scope is to retrieve the email address of the user.
-        // phone scope is to retrieve the telephone number of the user.
+        // address scope is to retrieve the country number of the user.
```

Comment:
Let's start the comment with a capital letter. Let's fix the other places as well.

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096